Update tutorial-pipeline-return-value.md (#117734)
Azure/azure-content: articles/data-factory/tutorial-pipeline-return-value.md

## Prerequisite - Calling a Child Pipeline

> The prerequisite of the use case, is that have an [Execute Pipeline Activity](control-flow-execute-pipeline-activity.md), calling a child pipeline. It's important that we enabled _Wait on Completion_ for the activity

Reviewer: The grammar isn't quite right.

```suggestion
As a prerequisite, your design needs an [Execute Pipeline Activity](control-flow-execute-pipeline-activity.md) calling a child pipeline, with _Wait on Completion_ enabled on the activity.
```
